src/config/bookconfig.rs

```diff
@@ -17,6 +18,8 @@ pub struct BookConfig {
     multilingual: bool,
     indent_spaces: i32,
 
+    playpen_config: PlaypenConfig,
```

Comment:
playpen is very much tied to the html version of a book, would it not be better to put the playpen config inside `HtmlConfig`?